moviebot/controller/controller.py

```diff
@@ -45,3 +48,46 @@ def restart(self, utterance: UserUtterance) -> bool:
             utterance: User utterance.
         """
         return utterance.get_text() == RESTART
+
+    def get_cursor(self) -> sqlite3.Cursor:
+        """Returns SQL cursor."""
+        conn = sqlite3.connect(self.configuration["DATA"]["db_path"])
+        c = conn.cursor()
+        return c
+
+    def load_user_data(self, path: str, user_id: str) -> Dict[str, Any]:
+        """Loads movie choices (accept/reject) for a user from conversation
+        history.
+
+        Args:
+            path: Path to conversation histories.
+            user_id: User id.
+
+        Returns:
+            Object with previous movie choices.
+        """
+        user_history_path = f"{path}user_{user_id}.json"
```

Comment:
Use `os.path.join` to construct the path. We do not want errors if users do not use a trailing `/` when defining path.

Reply:
Good catch, thanks. I changed to use `os.path.join`.